src/actions/plugins/easymotion/types.ts

```diff
@@ -11,4 +12,5 @@ export interface EasyMotionCharMoveOpions extends EasyMotionMoveOptionsBase {
 
 export interface EasyMotionWordMoveOpions extends EasyMotionMoveOptionsBase {
   labelPosition?: LabelPosition;
+  jumptoanywhere?: jumptoanywhere;
```

Comment:
Please camel case. Unless is it generally known as `jumptoanywhere` all lowercase?

Reply:
So all lowercase `jumptoanywhere` is how the original easymotion creators describe it in their vim documentation, so I was just trying to be consistent with the original implementation:
<img width="608" alt="screen shot 2018-03-23 at 12 09 10 pm" src="https://user-images.githubusercontent.com/4033621/37848987-3bd6b744-2e93-11e8-925a-947ad5f06840.png">

However, I agree that the camelcase naming convention is much easier to read and more JS-like. I have went ahead and updated all JS code references to be camelcased named, and only left the original `jumptoanywhere` text as documentation to describe the original matching motion found in standard VIM.